test/objgrad-on-tron.jl

```diff
@@ -7,7 +7,6 @@
   function MyProblem()
     meta = NLPModelMeta{Float64, Vector{Float64}}(
       2, # nvar
-      x0 = [0.1; 0.1],
```

Comment:
I wouldn't delete that one. The `x0` in the problem will actually be used as initial guess in the solvers.